core/arch/aarch64/emit_utils.c

```diff
@@ -690,8 +690,11 @@ emit_indirect_branch_lookup(dcontext_t *dc, generated_code_t *code, byte *pc,
                                                TLS_MASK_SLOT(ibl_code->branch_type),
                                                OPSZ_16)));
     /* and x1, x1, x2 */
-    APP(&ilist, XINST_CREATE_and(dc, opnd_create_reg(DR_REG_X1),
-                                 opnd_create_reg(DR_REG_X2)));
+    APP(&ilist, INSTR_CREATE_and_shift(dc, opnd_create_reg(DR_REG_X1),
```

Comment:
I think that AArch64 should follow what we did for ARM: make simpler convenience macros for the common cases that do not require specifying all these extra operands.  We have INSTR_CREATE_and() which automagically adds the DR_SHIFT_NONE by 0 and passes to INSTR_CREATE_and_shimm().